### PR TITLE
Add modulebuilder

### PR DESCRIPTION
**Pull request by sutser**

### CODE REVIEW

File: CompositionRoot/ModuleRegistrar.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Immutable;
+using Autofac;
+
+namespace Messerli.CompositionRoot
+{
+    public class ModuleRegistrar : Module
```
Comment (bash) [inline CompositionRoot/ModuleRegistrar.cs]: This class can be `internal sealed`

File: CompositionRoot/ModuleRegistrar.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        private readonly ImmutableList<Register> _mockRegistrations;
```
Comment (bash) [inline CompositionRoot/ModuleRegistrar.cs]: Why is this called `_mockRegistrations`?

File: CompositionRoot.Test/ModuleBuilderTest.cs [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+using Autofac.Core.Registration;
+using Messerli.CompositionRoot.Test.Stubs;
+using Xunit;
+
+namespace Messerli.CompositionRoot.Test
+{
+    public class ModuleBuilderTest
+    {
+        [Fact]
+        public void CanEmptyModuleBuilds()
```
Comment (bash) [inline CompositionRoot.Test/ModuleBuilderTest.cs]:
```suggestion
        public void CanBuildEmptyModule()
```

File: CompositionRoot/ModuleRegistrar.cs (hunk 2 above)
Comment (sutser) [reply CompositionRoot/ModuleRegistrar.cs]: Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/4/commits/43a7810c2cda586de25689d160d598303166a461

File: CompositionRoot/ModuleRegistrar.cs (hunk 1 above)
Comment (sutser) [reply CompositionRoot/ModuleRegistrar.cs]: Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/4/commits/91d57120fbae0705c7c53cc9d129d8293abedc96

Comment (sutser) [reply CompositionRoot/ModuleRegistrar.cs]: Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/4/commits/f3adbe9ed8bd578935367eee0c033a8c3ebd359b

**Pull request merged**